packages/server/lib/browsers/utils.ts

```diff
@@ -434,32 +434,36 @@ const initializeCDP = async (criClient: CriClient, automation: Automation) => {
 }
 
 const overrideServiceWorkerRegistration = (binding) => {
-  const oldRegister = window.ServiceWorkerContainer.prototype.register
+  // The service worker container won't be available in different situations (like in the placeholder iframes we create
+  // in open mode). So only override the register function if it exists.
+  if (window.ServiceWorkerContainer) {
```

Comment:
nice! actually came across this yesterday...
https://github.com/cypress-io/cypress/assets/14099737/dee9f272-b44b-4046-8070-e4743f358e9a